packages/calculator-bn.js/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@dinero.js/calculator-bn.js",
+  "version": "1.0.0-alpha.0",
+  "description": "BN.js calculator implementation for Dinero.js",
+  "keywords": [
+    "money",
+    "monetary",
+    "amount",
+    "immutable",
+    "calculator",
+    "bn.js"
+  ],
+  "homepage": "https://v2.dinerojs.com",
+  "bugs": "https://github.com/dinerojs/dinero.js/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dinerojs/dinero.js.git"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Exodus Movement, Inc.",
+    "url": "https://exodus.com"
+  },
+  "sideEffects": false,
+  "main": "dist/cjs/index.js",
+  "umd:main": "dist/umd/index.production.js",
+  "jsdelivr": "dist/umd/index.production.js",
+  "unpkg": "dist/umd/index.production.js",
+  "module": "dist/esm/index.js",
+  "source": "src/index.ts",
+  "types": "dist/esm/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:clean": "rimraf ./{dist,temp}",
+    "build:esm": "babel src --root-mode upward --extensions '.ts' --out-dir dist/esm --ignore '**/*/__tests__/'",
+    "build:tsc": "tsc -b",
+    "build:types": "yarn run build:tsc && api-extractor run --local",
+    "build:umd:cjs": "rollup --config",
+    "build": "yarn build:clean && yarn build:umd:cjs && yarn build:esm && yarn build:types",
+    "on:change": "concurrently \"yarn build:esm\" \"yarn build:types\"",
+    "prepare": "yarn build:esm && yarn build:types",
+    "watch": "watch \"yarn on:change\" --ignoreDirectoryPattern \"/dist/\""
+  },
+  "dependencies": {
+    "@dinero.js/core": "2.0.0-alpha.12",
+    "bn.js": "^5"
```

Comment:
This should be an exact version

Reply:
maybe a peerDependency instead? There's two ways it'll work with dinero - by passing BN instances into `amount`, and by calculator-bn using it internally. Ideally those are both supported in a similar way

Reply:
Adding it as a peerDependency could solve the issue with rollup too.

Reply:
Though, I think it then needs to be added to  `devDependencies` in the root `package.json`

Reply:
yep, already added